src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImplTest.java

```diff
@@ -72,19 +77,34 @@ void setup() {
     void processMessageTest() throws ServiceException {
         ChipsRestInterfacesSend chipsRestInterfacesSend = new ChipsRestInterfacesSend();
 
-        messageProcessorService.processMessage(chipsRestInterfacesSend);
+        messageProcessorService.processMessage(chipsRestInterfacesSend, false);
 
         verify(chipsRestClient, times(1)).sendToChips(eq(chipsRestInterfacesSend));
         verify(messageProducer, times(0)).writeToTopic(any(), eq(RETRY_TOPIC));
         verify(messageProducer, times(0)).writeToTopic(any(), eq(ERROR_TOPIC));
     }
 
+    @Test
+    void processErrorMessageTest() throws ServiceException {
+        ChipsRestInterfacesSend chipsRestInterfacesSend = new ChipsRestInterfacesSend();
+        RuntimeException runtimeException = new RuntimeException("runtimeException");
+        doThrow(runtimeException).when(chipsRestClient).sendToChips(chipsRestInterfacesSend);
+        when(consumerConfig.retryTopic()).thenReturn(RETRY_TOPIC);
+
+        messageProcessorService.processMessage(chipsRestInterfacesSend, true);
+
+        verify(chipsRestClient, times(1)).sendToChips(eq(chipsRestInterfacesSend));
```

Comment:
minor thing but you should be able to remove eq() if you don't use any other matchers in the args.